src/_util/dom.ts

```diff
@@ -202,13 +202,22 @@ export const getAttach = (node: any): HTMLElement => {
 };
 
 // 获取 css vars
-export const getCssVarsValue = (name: string, element?: HTMLElement) => {
-  if (!canUseDocument) return;
+export const getCssVarsValue = (name: string, element?: HTMLElement): string | undefined => {
```

Comment:
```suggestion
export const getCssVarsValue = (name: string, element?: HTMLElement) => {
```
一定需要主动声明类型吗，有什么具体的需求吗，ts 在这里能主动推断的吧？

Reply:
这里推断的是string，都没有undefined的情况

Reply:
但是在 206 出现了 undefined，return 在没有指定变量的时候返回的是 undefined。

https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABMOcAUEEBMBciBGqANgKYCGYAlIgN4BQijiA9M0wHoD8DTMwiGbNQBOJKCGFgA3D0ajxkxAEYZAXyA
<img width="499" alt="image" src="https://github.com/Tencent/tdesign-react/assets/51358815/8a993ede-4f9a-436c-b78b-816ecf386794">

Reply:
<img width="869" alt="image" src="https://github.com/Tencent/tdesign-react/assets/88650465/421efe48-c44a-4182-b5de-dc07cf28de9a">
我不知道是不是ts版本的问题

Reply:
<img width="462" alt="image" src="https://github.com/Tencent/tdesign-react/assets/51358815/9f4053e5-44e6-45f2-b6bb-f1bd08a062b7">

Reply:
这个是另外的问题，并不是 undefined 导致的，可以直接按照 parseFloat 的修改方法处理一下就好了。待会儿我看看

Reply:
行！所以我尝试主动声明类型了